.ldrelease/config.yml

```diff
@@ -8,11 +8,11 @@ publications:
 
 jobs:
   - docker:
-      image: gradle:6.8.3-jdk11
+      image: gradle:7.3.3-jdk8
     template:
       name: gradle
-    skip:
-      - test  # don't try to run unit tests in the Releaser environment, since they require Redis
+      skip:
+        - test  # don't try to run unit tests in the Releaser environment, since they require Redis
```

Comment:
This indentation change is deliberate - the file was wrong before (but Releaser wasn't so strict about validation before, so the property was being ignored).